packages/tools/replay-tool/package.json

```diff
@@ -32,7 +43,7 @@
 		"lint:fix": "npm run prettier:fix && npm run eslint:fix",
 		"prettier": "prettier --check . --cache --ignore-path ../../../.prettierignore",
 		"prettier:fix": "prettier --write . --cache --ignore-path ../../../.prettierignore",
-		"tsc": "tsc"
+		"tsc": "fluid-tsc commonjs --project ./tsconfig.cjs.json && copyfiles -f ../../../common/build/build-common/src/cjs/package.json ./dist"
```

Comment:
Probably an easy conversion to ESM if downstream snapshot tests is converted.

Reply:
Can we convert both to ESM only at same time?